src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentServer.kt

```diff
@@ -45,7 +45,7 @@ interface CodyAgentServer {
   @JsonRequest("featureFlags/getFeatureFlag")
   fun evaluateFeatureFlag(flagName: GetFeatureFlag): CompletableFuture<Boolean?>
 
-  @JsonRequest("graphql/currentUserIsPro") fun isCurrentUserPro(): CompletableFuture<Boolean?>
+  @JsonRequest("graphql/currentUserIsPro") fun isCurrentUserPro(): CompletableFuture<Boolean>
```

Comment:
what if the agent `return null`? is it possible?

Reply:
We can get null from an agent but If we get null then `isCurrentUserPro != true` I guess both are fine i dont really care 🤷   
https://github.com/sourcegraph/jetbrains/pull/507#discussion_r1481033113